en/organisation-standard/overview/organisations.rst

```diff
@@ -1,6 +1,15 @@
 Type of Organisations
 =====================
 
+The new activity participants guidance can be found here:
+
+- `Activity Participants <https://drive.google.com/file/d/1TI9PU5dyyRE2IzervFPUZz_Xfue16xYC/view?usp=sharing>`__: explains how to refer to and find different organisations within the organisation file.
```

Comment:
This is the URL for "Related Documents".

Reply:
Good spot Andy